render/src/main/java/org/codice/imaging/nitf/render/imagerep/Rgb48ImageRepresentationHandler.java

```diff
@@ -20,10 +20,23 @@
 
 import javax.imageio.stream.ImageInputStream;
 
+import org.codice.imaging.nitf.core.image.PixelJustification;
+
 class Rgb48ImageRepresentationHandler extends AbstractRgbImageRepresentationHandler {
+    private final int bitsToDiscard;
+
     Rgb48ImageRepresentationHandler(final Map<Integer, Integer> bandMap,
-            final int actualBitsPerPixelPerBand) {
+            final int actualBitsPerPixelPerBand, final PixelJustification pixelJustification) {
         super(bandMap, actualBitsPerPixelPerBand);
+        if (actualBitsPerPixelPerBand - Byte.SIZE < 0) {
+            this.bitsToDiscard = 0;
```

Comment:
According to MIL-STD-2500A Table III, the ABPP and NBPP can both be between 1-64. Not sure if this conditional actually ever happens, but the specification seems to allow for it. That's a really good point though: if this conditional is met, we'll need to shift based on the pixel justification. I will put in the fix and see if I can write up a unit test that tests this case.